manim/utils/color/core.py

```diff
@@ -18,6 +18,27 @@
 
     The colors of type "C" have an alias equal to the colorname without a letter,
     e.g. GREEN = GREEN_C
+
+===================
+Custom Color Spaces
+===================
+
+Hello dear visitor, you seem to be interested in implementing a custom color class for a color space we don't currently support.
+
+The current system is using a few indirections for ensuring a consistent behavior with all other color types in manim.
+
+To implement a custom color space you must subclass :class:`ManimColor` and implement three important functions
+
+:func:`~.ManimColor._internal_value` is an ``@property`` implemented on :class:`ManimColor` with the goal of keeping a consistent internal representation that can be referenced by other functions in :class:`ManimColor`.
```

Comment:
```suggestion
:attr:`~.ManimColor._internal_value` is an ``@property`` implemented on :class:`ManimColor` with the goal of keeping a consistent internal representation that can be referenced by other functions in :class:`ManimColor`.
```
Oops, missed this before. Hopefully this allows sphinx to hyperlink it.